servers/partner-ci/9.x/nsxv.yaml

```diff
@@ -286,11 +288,20 @@
       - test-property
     builders:
       - shell: |
```

Comment:
let's create a marcos or yaml anchors to combine steps like that to avoid code duplication